tools/slurm/inference.slurm.sh

```
@@ -0,0 +1,39 @@
+
+set -uo pipefail
+
+echo "===== SIMBA Inference ====="
+echo "Job ID: $SLURM_JOB_ID"
+echo "Node:   $SLURM_NODELIST"
+echo "Date:   $(date)"
+
+PREPRO_DIR=/mnt/data2/nkubrakov/massspecgym/preprocessing_th20_asb
+CHECKPOINT_DIR=/mnt/data2/nkubrakov/massspecgym/checkpoints_th20_asb
+
+cd /home/nkubrakov/simba
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)